site/docs/advanced-transforms.mdx

```diff
@@ -32,12 +32,12 @@ about your CSS in order for the transform to be safe. Most of these transforms
 fall into either or both of categories.
 
 ### Assumes concatenation
-
+12345678901234567890123456789012345678901234567890123456789012345678901234567890
```

Comment:
What do these numbers mean? I think they should be removed.

Reply:
I have no idea what they are. Removed.